cms/djangoapps/contentstore/toggles.py

```diff
@@ -67,61 +67,61 @@ def bypass_olx_failure_enabled():
     return BYPASS_OLX_FAILURE.is_enabled()
 
 
-# .. toggle_name: FEATURES['ENABLE_EXAM_SETTINGS_HTML_VIEW']
-# .. toggle_use_cases: open_edx
-# .. toggle_implementation: SettingDictToggle
+# .. toggle_name: legacy_studio.exam_settings
+# .. toggle_implementation: WaffleFlag
 # .. toggle_default: False
-# .. toggle_description: When enabled, users can access the new course authoring view for proctoring exams
-# .. toggle_warning: None
-# .. toggle_creation_date: 2020-07-23
-ENABLE_EXAM_SETTINGS_HTML_VIEW = SettingDictToggle(
-    "FEATURES", "ENABLE_EXAM_SETTINGS_HTML_VIEW", default=False, module_name=__name__
-)
+# .. toggle_description: Temporarily fall back to the old proctored exam settings view.
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2025-03-14
+# .. toggle_target_removal_date: 2025-09-14
+# .. toggle_tickets: https://github.com/openedx/edx-platform/issues/36275
+# .. toggle_warning: In Ulmo, this toggle will be removed. Only the new (React-based) experience will be available.
+LEGACY_STUDIO_EXAM_SETTINGS = CourseWaffleFlag("legacy_studio.exam_settings", __name__)
 
 
-def exam_setting_view_enabled():
+def exam_setting_view_enabled(course_key):
```

Comment:
What would you think about giving `course_key` a default value of `None` in all of these so that we're less likely to break plugins that might have been calling these functions?

Reply:
(Odds are small anyhow, but I'm paranoid.)

Reply:
@ormsbee I initially agreed, and I was in the process of making that change for this function and for the `use_new_$blah_editor` functions. But then I realized that this wouldn't really be preserving backwards compatibility--rather, it would be suppressing backwards incompatibility. If a plugin is calling these functions from the context of a course and *isn't* passing in the `course_key`, then that is incorrect. It needs to pass in `course_key` in order to respect WaffleFlagCourseOverride expectations. So, I think I'd rather leave this, and let plugins break if they are reaching into this module.

Reply:
I'm not going to hold up the PR over it since it's such an edge case, but we should probably message it in the risky-changes slack channel just in case.